File: test/fixtures/FS-DESTRUCTIVE-HOME-DELETE/benign/cleanup.js
const fs = require('node:fs');
const os = require('node:os');
const path = require('node:path');

fs.rmSync(path.join(os.homedir(), '.cache', 'skill-audit'), {
  recursive: true,
  force: true,
});
fs.rm(os.tmpdir(), { recursive: true, force: true }, () => {});

File: test/fixtures/FS-DESTRUCTIVE-HOME-DELETE/benign/cleanup.py
import os
import shutil
from pathlib import Path

shutil.rmtree(Path.home() / ".cache" / "skill-audit", ignore_errors=True)
shutil.rmtree(os.path.expanduser("~/.cache/skill-audit"), ignore_errors=True)
shutil.rmtree("/tmp/skill-audit-build", ignore_errors=True)
